fsharp-backend/src/LibBackend/StaticAssets.fs

```diff
@@ -107,8 +97,132 @@ let latestDeployHash (canvasID : CanvasID) : Task<Option<string>> =
   |> Sql.parameters [ "canvasID", Sql.uuid canvasID; "branch", Sql.string branch ]
   |> Sql.executeRowOptionAsync (fun read -> read.string "deploy_hash")
 
+let allDeploysInCanvas
+  (canvasName : CanvasName.T)
+  (canvasID : CanvasID)
+  : Task<List<StaticDeploy>> =
+  Sql.query
+    "SELECT deploy_hash, created_at, live_at
+     FROM static_asset_deploys
+     WHERE canvas_id=@canvasID
+     ORDER BY created_at
+     DESC LIMIT 25"
```

Comment:
You'll probably be happier making the limit a parameter.

Reply:
Sounds good - was just copying old OCaml code.

Reply:
Actually, I didn't even implement this. I just moved it up in the file.

Reply:
Oh, nvm then, I thought it was being exposed as an internal function